spec/helpers/csv_helper.rb

```diff
@@ -12,7 +12,7 @@
 
 module GutData::Helpers
   module CsvHelper
-    CSV_PATH_EXPORT = 'out.txt'
+    CSV_PATH_EXPORT = 'tmp/out.txt'
```

Comment:
just curious, why did this matter?

Reply:
Running the tests produced temporary files, so it made sense to move them to a temporary folder so that they would not be tracked or alternatively we could have added them to the gitignore. This was the main takeaway. I got after chatting with the Andrews™

Reply:
When we ran the test suite there would be a few files that were dumped into the root directory: this and `deprecations.txt`. They were untracked files and kept getting picked up by git. So instead, I just changed them to pipe to the `tmp/` instead of the project root.

Adding those two to `.gitignore` would effectively have had the same effect